libs/remix-ui/app/src/lib/remix-app/components/modals/enter.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useContext, useEffect, useState} from 'react'
+import {UsageTypes} from '../../types'
+import { type } from 'os'
+
+declare global {
+  interface Window {
+    _paq: any
+  }
+}
+const _paq = (window._paq = window._paq || [])
+
+interface EnterDialogProps {
+  hide: boolean,
+  handleUserChoice: (userChoice: UsageTypes) => void,
+}
+
+const EnterDialog = (props: EnterDialogProps) => {
+  const [visibility, setVisibility] = useState<boolean>(false)
+
+  useEffect(() => {
+    setVisibility(!props.hide)
+  }, [props.hide])
+
+  const enterAs = async (uType) => {
+    props.handleUserChoice(uType)
+
+    switch (uType) {
+    case UsageTypes.Beginner: {
+      _paq.push(['trackEvent', 'enterDialog', 'usageType', 'beginner'])
+      break
+    }
+    case UsageTypes.Tutor: {
+      _paq.push(['trackEvent', 'enterDialog', 'usageType', 'tutor'])
+      break
+    }
+    case UsageTypes.Prototyper: {
+      _paq.push(['trackEvent', 'enterDialog', 'usageType', 'prototyper'])
+      break
+    }
+    case UsageTypes.Production: {
+      _paq.push(['trackEvent', 'enterDialog', 'usageType', 'production'])
+      break
+    }
+    default: throw new Error()
+    }
+  }
+
+  const modalClass = !visibility ? "d-none" : "d-flex"
+  return (
+    <div
+      data-id={`EnterModalDialogContainer-react`}
+      data-backdrop="static"
+      data-keyboard="false"
+      className={"modal  " + modalClass}
+      role="dialog"
+    >
+      <div className="modal-dialog align-self-center pb-4" role="document">
+        <div
+          tabIndex={-1}
+          className={'modal-content remixModalContent mb-4'}
+          onKeyDown={({keyCode}) => {
+          }}
+        >
+          <div className="modal-header d-flex flex-column">
+            <h3 className='text-dark'>Welcome to Remix IDE</h3>
+            <div className='d-flex flex-row pt-2'>
+              <h6 className="modal-title" data-id={`EnterModalDialogModalTitle-react`}>
+                To load the project with the most efficiant setup we would like to know your experiance type.
```

Comment:
efficient not efficiant

Reply:
Suggestions for text:

(If for usage stats only):
To improve the Remix user experience, we would like to know the most useful setup for you:

(if UI will actually load differently):
To start Remix with the most useful setup, please select your primary workflow:

Buttons:
"Learn/Playground"... loads LearnETH and HelloWorld (or whatever template is used in LearnETH)
"Teach/Workshop"... loads blank or just pragma header and filename field or specific template
"Prototype/Test"... loads blank workspace and filename field
"Deploy/Production"... loads normal UI for "Clone" and "Recent Workspaces" fields

Consider some possible alternates...
"Learn Solidity"... loads LearnETH and HelloWorld (or whatever template is used in LearnETH)
"Prototype/Code"... loads blank workspace and filename field
"Test/Debug"... loads "Clone" etc, and debugger and SUT plugin
"Deploy/Production"... loads normal UI for "Clone" and "Recent Workspaces" fields

Reply:
Deploy can be missleading. beginners also want to do it

Reply:
I think I like the first 3 but not with deploy.

Also keep in mind we can add some details the tooltips